eng/testing/.runsettings

```diff
@@ -1,35 +1,2 @@
-<?xml version="1.0" encoding="utf-8"?>
-<RunSettings>
-  <RunConfiguration>
-    <!-- Timeout in ms, 20 minutes -->
-    <TestSessionTimeout>1200000</TestSessionTimeout>
-    <!-- Filter out failing (wrong framework, platform, runtime or activeissue) tests -->
-    <TestCaseFilter>category!=failing</TestCaseFilter>
-  </RunConfiguration>
-  <LoggerRunSettings>
-    <Loggers>
-      <Logger friendlyName="trx">
-        <Configuration>
-          <LogFileName>TestResults.trx</LogFileName>
-        </Configuration>
-      </Logger>
-      <Logger friendlyName="console">
-        <Configuration>
-          <Verbosity>normal</Verbosity>
-        </Configuration>
-      </Logger>
-      <Logger friendlyName="blame" enabled="True" />
-    </Loggers>
-  </LoggerRunSettings>
-  <DataCollectionRunSettings>
-    <DataCollectors>
-      <!-- Enables blame -->
-      <DataCollector friendlyName="blame" enabled="True">
-        <Configuration>
-          <CollectDump DumpType="Full" />
-          <CollectDumpOnTestSessionHang TestTimeout="7min" HangDumpType="Full" />
-        </Configuration>
-      </DataCollector>
-    </DataCollectors>
-  </DataCollectionRunSettings>
-</RunSettings>
+# Filter out failing (wrong framework, platform, runtime or activeissue) tests
+--filter-not-trait "category=failing"
```

Comment:
how is eng/testing/.runsettings discovered if there's no tests below it?

Reply:
tests/Directory.Build.props has:

```xml
    <RunSettingsFilePath>$(RepositoryEngineeringDir)testing\.runsettings</RunSettingsFilePath>
```

That's then used in:

```xml
    <TestingPlatformCommandLineArguments Condition="'$(RunSettingsFilePath)'!=''">$(TestingPlatformCommandLineArguments) @"$(RunSettingsFilePath)"</TestingPlatformCommandLineArguments>
```